addons/medical_damage/stringtable.xml

```diff
@@ -687,5 +687,13 @@
             <Polish>Ustawia próg obrażeń jakie może otrzymać jednostka przed utratą przytomności. (0 jako ustawienie domyślne misji)</Polish>
             <Turkish>Bilinçsiz duruma geçmeden önce bir birimin alabileceği hasar miktarını ayarlar. (Görev varsayılanı için 0)</Turkish>
         </Key>
+        <Key ID="STR_ACE_Medical_Damage_PainUnconsciousChance_DisplayName">
+            <English>Pain Unconscious Chance</English>
+            <Polish>Szansa na nieprzytomność przez ból</Polish>
+        </Key>
+        <Key ID="STR_ACE_Medical_Damage_PainUnconsciousChance_Description">
+            <English>The probability of a person falling unconscious when their pain is above the tolerance threshold.</English>
```

Comment:
This is not completely accurate. The chance is not to be knocked out when you are just over the pain threshold. It is the chance to be knocked when you receive damage and if then you go over the tolerance threshold. It is checked only then. So accurate description would be:

"Chance of being knocked unconscious when receiving damage and exceeding pain threshold"

Reply:
Just add "when receiving damage". If we want to change "probability" to "chance", that should be done in separate PR for all settings. But I don't see why.